lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -118,7 +118,10 @@ e.g. 'tag:0123:important:GM' returns a list of '0123', 'important' and 'GM'.
 =cut
 sub tag {
     my ($self) = @_;
-    $self->text =~ /\btag:(?<build>([-.@\d\w]+-)?[@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    $self->text =~ /\btag:(?<build>[-.@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    # note: calling this 'build' is a slight lie, because it may be
```

Comment:
yeah, fine. We couldn't find a better name for "version-build" so we just continued to call it build and that is how we introduced this bug here.